app/src/hooks/useGlobalState.ts

```diff
@@ -2,4 +2,4 @@ import { TypedUseSelectorHook, UseSelector, useDispatch, useSelector } from "rea
 import { AppDispatch, RootState } from "../types/redux.types";
 
 export const useAppDispatch = () => useDispatch<AppDispatch>();
-export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector as UseSelector;
\ No newline at end of file
+export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector<RootState>;
```

Comment:
as I understand this typization is more clear? right?

Reply:
tg